docs/examples/sujoy_edge/sujoy_edge_demo.jl

```diff
@@ -0,0 +1,80 @@
+# ---
+# cover: assets/sedge.png
+# title: Sujoy Edge Demo
+# ---
+
+using Images,Statistics,ImageView,TestImages
+
+# Sujoy edge function
+
+
+# [Paper Link] (https://www.ijert.org/research/a-better-first-derivative-approach-for-edge-detection-IJERTV2IS110616.pdf)
```

Comment:
Would you mind to give some insightful explanation here on what Sujoy function is about. I mean, some description about it. It could a paragraph, or just one or two sentences.

Reply:
Fix this